src/components/image/index.tsx

```diff
@@ -63,14 +65,16 @@ export function Image({
     type: "url",
   });
 
-  const { value } = values;
-  let { error: thingError } = values;
-
+  const { value, thing, error: thingError } = values;
+  let valueError;
   if (!edit && !value) {
-    thingError = new Error("No value found for property.");
+    valueError = new Error("No value found for property.");
   }
+  const isFetchingThing = !thing && !thingError;
 
-  const [error, setError] = useState<Error | undefined>(thingError);
+  const [error, setError] = useState<Error | undefined>(
+    thingError || valueError
```

Comment:
They're not booleans, so `??` checks whether this is actually `undefined` or `null` rather than just any falsy value.

```suggestion
    thingError ?? valueError
```